srp/Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [package]
 name = "srp"
 version = "0.3.0"
+edition = "2018"
```

Comment:
Using 2018 edition requires Rust 1.31, so with current [policy](https://github.com/RustCrypto/hashes#minimum-supported-rust-version-msrv) we have to bump minor version.

Reply:
yeah, I used `cargo-release` for SPAKE2 releases, and I figure we can use it for the next SRP release too. I guess we need a note to remind us to run it with `minor` instead of `patch` for both crates.

I don't have a whole lot of experience with `cargo-release`, do you know, if we set the SRP version to `0.4.0-alpha.0`, would that tell `cargo-release patch` to use `0.4.0`? It looks like the post-release version on SPAKE2 behaved that way..

Reply:
Ah, ok then. Would you mind publishing update for `srp` as well then? You should have necessary rights. BTW it would be nice if you'll add `spake2` publishing rights for @RustCrypto/pakes group, you can do it with the following command: `cargo owner --add github:RustCrypto:PAKEs`. (IIUC it will allow only to publish and yank crates, but not edit owners)

>do you know, if we set the SRP version to 0.4.0-alpha.0, would that tell cargo-release patch to use 0.4.0?

Unfortunately, no. I still prefer doing things manually, though I consider moving to `cargo release` eventually. :)